scripts/add-cluster.sh

```diff
@@ -128,6 +142,9 @@ rules:
   - get
   - list
   - watch
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get", "list"]
```

Comment:
could you please update the format here?

Reply:
Done ! please see [7ee45c5](https://github.com/codeready-toolchain/toolchain-cicd/pull/88/commits/7ee45c5f61a37af5e3ba1b281b824073d8c006a9)

Sorry, I did it in sandbox-sre and forgot to apply same format here! Thanks for pointing that out!